### PR TITLE
getting data from DB to appear in portfolio front-end 

### PR DESCRIPTION
**Pull request by snack-attack**

### CODE REVIEW

File: projects.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+require_once './database.php';
+
```
Comment (alicemcneil) [inline projects.php:3]: you could refactor this into functions eventually :)

**Review by alicemcneil — COMMENTED** [review]
looks good! could potentially be made more concise eventually? :)

File: projects.php (hunk 1 above)
Comment (snack-attack) [reply projects.php:3]: Will do!

File: database.php [hunk 2]
```diff
@@ -1,15 +1,10 @@
 <?php
 
-$db_server = 'mysql:host=192.168.20.20';
-$db_username = 'root';
-$db_password = '';
-$db_database = 'portfolio-cms';
+// $db_server = 'mysql:host=192.168.20.20';
```
Comment (sjr-mayden) [inline database.php:3]: Delete commented code before merging

File: projects.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        JOIN `tag` ON `tag_id`=`tag`.`id`;";
+$query = $db->prepare($sql);
+$query->execute();
+$project_tags = $query->fetchAll();
+
+
+foreach($projects as $project) {
+    echo '<div class="';
+    //alternate classes so the card style switches every other project
+    if ($project['id'] % 2 == 0) {
```
Comment (sjr-mayden) [inline projects.php:27]: Fine as long as the projects you're going to display are sequential/fit the pattern odd, even, odd etc.

Comment (sjr-mayden) [reply projects.php:27]: Suspect this might not be the case in the future

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: projects.php (hunk 3 above)
Comment (snack-attack) [reply projects.php:27]: Agreed, I'll probably need to refactor to a better solution. It's offically added to my roadmap!

**Pull request merged**